docs/customization.md

```diff
@@ -0,0 +1,110 @@
+
+## Customizing the UI
+
+The frontend is built using [React](https://reactjs.org/) and [Fluent UI components](https://react.fluentui.dev/). The frontend components are stored in the `app/frontend/src` folder. The typical components you'll want to customize are:
+
+- `app/frontend/index.html`: To change the page title
+- `app/frontend/src/pages/layout/Layout.tsx`: To change the header text and logo
+- `app/frontend/src/pages/chat/Chat.tsx`: To change the large heading
+- `app/frontend/src/components/Example/ExampleList.tsx`: To change the example questions
+
+## Customizing the backend
+
+The backend is built using [Quart](https://quart.palletsprojects.com/), a Python framework for asynchronous web applications. The backend code is stored in the `app/backend` folder.
+
+### Chat/Ask approaches
+
+Typically, the primary backend code you'll want to customize is the `app/backend/approaches` folder, which contains the classes powering the Chat and Ask tabs. Each class uses a different RAG (Retrieval Augmented Generation) approach, which include system messages that should be changed to match your data
+
+#### Chat approach
+
+The chat tab uses the approach programmed in [chatreadretrieveread.py](https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/app/backend/approaches/chatreadretrieveread.py).
+
+1. It uses the ChatGPT API to turn the user question into a good search query.
+2. It queries Azure Cognitive Search for search results for that query (optionally using the vector embeddings for that query).
+3. It then combines the search results and original user question, and asks ChatGPT API to answer the question based on the sources. It includes the last 4K of message history as well (or however many tokens are allowed by the deployed model).
+
+The `system_message_chat_conversation` variable is currently tailored to the sample data since it starts with "Assistant helps the company employees with their healthcare plan questions, and questions about the employee handbook." Change that to match your data.
+
+### Ask approach
+
+The ask tab uses the approach programmed in [retrievethenread.py](https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/app/backend/approaches/retrievethenread.py).
+
+1. It queries Azure Cognitive Search for search results for the user question (optionally using the vector embeddings for that question).
+2. It then combines the search results and user question, and asks ChatGPT API to answer the question based on the sources.
+
+The `system_chat_template` variable is currently tailored to the sample data since it starts with "You are an intelligent assistant helping Contoso Inc employees with their healthcare plan questions and employee handbook questions." Change that to match your data.
+
+## Improving answer quality
```

Comment:
@charris-msft Section we talked about today

Reply:
LGTM